tools/ratbagd.py

```diff
@@ -275,12 +275,27 @@ def devices(self):
         """A list of RatbagdDevice objects supported by ratbagd."""
         return self._devices
 
+    def __getitem__(self, id):
+        """Returns the requested device, or None."""
+        dev = None
+        for d in self.devices:
+            if d.id == id:
+                dev = d
+                break
+        return dev
```

Comment:
Why not the simpler equivalent below?

```python
for d in self.devices:
    if d.id == id:
        return d
return None
```

Reply:
... Because I just copy pasted the code without giving more than 2 seconds of thinking :)

I'll update and push a new version.